stable/traefik/templates/deployment.yaml

```diff
@@ -36,6 +36,9 @@ spec:
         chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
         release: {{ .Release.Name | quote }}
         heritage: {{ .Release.Service | quote }}
+{{- if .Values.deployment.podLabels }}
+{{ toYaml .Values.deployment.podLabels | indent 8 }}
+{{- end }}
```

Comment:
Please indent line 39 and 41 the 8 spaces so that this if statement follows the style of the rest of the code.

Line 40, needs to remain unindented, though, as you've emphasized.